Lib/webbrowser.py

```diff
@@ -15,14 +15,15 @@ class Error(Exception):
 
 _browsers = {}          # Dictionary of available browser controllers
 _tryorder = []          # Preference order of available browsers
+_preferred_browser = None # The preferred browser for the current environment
 
-def register(name, klass, instance=None, update_tryorder=1):
-    """Register a browser connector and, optionally, connection."""
+def register(name, klass, instance=None, priority=False):
```

Comment:
I took the opportunity to make what I thought was an improvement. The tri-state definition of the old parameter seemed a bit clumsy and unnecessary (note that it was left out of the documentation).

I can back out the change to update_tryorder if it is deemed appropriate.